README.md

```diff
@@ -10,6 +10,19 @@ Install it as a global tool here:
 dotnet tool install -g try-convert
 ```
 
+To convert everything in your solution run this:
```

Comment:
I think a preferred approach is just to run `try-convert` without any addition arguments in the root directory of a solution or project. You can use `-w` but it's not required if the directory you're in has a solution file.